config/dev.exs

```diff
@@ -33,12 +33,12 @@ config :teslamate, TeslaMate.Repo,
   username: "postgres",
   password: "postgres",
   database: "teslamate_dev",
-  hostname: "localhost",
+  hostname: System.get_env("DATABASE_HOST", "localhost"),
```

Comment:
This is no longer necessary with the latest changes. The env variables are now respected in development too! :-)